configuration/configuration.go

```diff
@@ -464,14 +468,31 @@ func (c *ConfigurationData) GetCacheControlUser() string {
 
 // GetTokenPrivateKey returns the private key (as set via config file or environment variable)
 // that is used to sign the authentication token.
-func (c *ConfigurationData) GetTokenPrivateKey() []byte {
-	return []byte(c.v.GetString(varTokenPrivateKey))
+func (c *ConfigurationData) GetTokenPrivateKey() (*rsa.PrivateKey, error) {
+	if c.tokenPrivateKey == nil {
```

Comment:
Do we have to protect this `c.tokenPrivateKey` in this "singleton"-type of creation?